src/main/java/uk/gov/hmcts/darts/arm/component/impl/AutomatedTaskProcessorFactoryImpl.java

```diff
@@ -120,13 +120,4 @@ public GenerateCaseDocumentForRetentionDateProcessor createGenerateCaseDocumentF
             throw new DartsException(String.format("Batch size not supported for generating case document for retention date: '%s'", batchSize));
         }
     }
-
-    @Override
-    public CloseOldCasesProcessor createCloseOldCasesProcessor(int batchSize) {
```

Comment:
If you're removing this one, should you not remove the other ones in this class for consistency? Assuming there was a genuine reason for removing this.